src/dataflow-types/src/client/controller.rs

```diff
@@ -133,9 +133,6 @@ impl Default for ClusterReplicaSizeMap {
 }
 
 /// Deterministically generates replica names based on inputs.
-///
-/// This function needs to be publicly accessible so other layers can ensure we
-/// do not attempt to create multiple replicas with the same name.
```

Comment:
I assume this comment was outdated, since the function is *not* publicly accessible.